inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplImportExportIntegrationTest.java

```diff
@@ -103,6 +103,7 @@ public void tearDown() throws Exception {
 
     @Test
     public void thatApplicationContextStarts() {
+        throw new UnsupportedOperationException();
```

Comment:
Since this is an empty test, wouldn't it be better to remove the test?

Reply:
Removed the Empty test.